src/apps/js_generic/js_generic.cpp

```diff
@@ -16,9 +21,13 @@ namespace ccfapp
 
   using Table = kv::Map<std::vector<uint8_t>, std::vector<uint8_t>>;
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wc99-extensions"
+
```

Comment:
Likewise.

Reply:
This one seems necessary because of the `JS_NULL` macro which we get from QuickJS.

Reply:
Ah yes!